pkgs/tools/networking/openssh/common.nix

```diff
@@ -45,10 +45,14 @@ stdenv.mkDerivation {
   ] ++ extraPatches;
 
   postPatch =
-    # On Hydra this makes installation fail (sometimes?),
-    # and nix store doesn't allow such fancy permission bits anyway.
     ''
+      # On Hydra this makes installation fail (sometimes?),
+      # and nix store doesn't allow such fancy permission bits anyway.
       substituteInPlace Makefile.in --replace '$(INSTALL) -m 4711' '$(INSTALL) -m 0711'
+
+      # Apply zlib-1.3 fix:
+      #   https://github.com/openssh/openssh-portable/commit/cb4ed12ffc332d1f72d054ed92655b5f1c38f621
+      substituteInPlace configure --replace 'if (n != 3 && n != 4)' 'if (n < 1)'
```

Comment:
Assume this is because we don't want to have to re-auto-gen everything.

Reply:
I was on the fence thinking of using `autoreconfHook` or not for a small change. I can see why a raw replacement is confusing. Switched to `autoreconfHook`.

Reply:
This is only until the next version is released, or am I mistaken here?

Reply:
Yes, next release will contain the fix.